packages/matchbox/src/components/Page/Page.js

```diff
@@ -11,16 +11,17 @@ import { Popover } from '../Popover';
 import { UnstyledLink } from '../UnstyledLink';
 import { ScreenReaderOnly } from '../ScreenReaderOnly';
 import { filterByVisible } from '../../helpers/array';
+import { tokens } from '@sparkpost/design-tokens';
 
 const StyledLink = styled(UnstyledLink)`
   text-decoration: none;
 `;
 
 function Breadcrumb({ content, ...rest }) {
   return (
-    <StyledLink {...rest} fontSize="200" lineHeight="200" fontWeight="medium">
-      <Box as="span" display="inline-flex" align-items="center">
-        <ChevronLeft size={20} />
+    <StyledLink {...rest} fontSize="200" lineHeight="450" fontWeight="medium">
```

Comment:
Found a bug here, UnstyledLink doesn't accept `fontSize="200" lineHeight="450" fontWeight="medium"` at all, I don't see the styles being applied.

Reply:
At least here it doesn't seem like it works: https://matchbox-storybook.netlify.app/?path=/story/layout-page--basic-example